packages/workspace/src/browser/workspace-commands.ts

```diff
@@ -152,6 +152,7 @@ export class FileMenuContribution implements MenuContribution {
             commandId: FileDownloadCommands.DOWNLOAD.id,
             order: 'b'
         });
+
```

Comment:
seems to be only whitespace change

Reply:
I see your point, but it was a change in the original code: https://github.com/theia-ide/theia/pull/4890/files#diff-7e70f039e1d4a1a1b544cf48997743e0R149

I did not want to change it, should I?

Reply:
you can change anything in your commits, but it is up to you, fine to merge like that as well